visualisation/volscope.cpp

```diff
@@ -0,0 +1,187 @@
+ *
+ **/
+/**
+ * @file volscope.cpp
+ * @ingroup visualization
+ * @author David Coeurjolly (\c david.coeurjolly@cnrs.fr )
+ * Laboratoire d'InfoRmatique en Image et Systèmes d'information - LIRIS (CNRS, UMR 5205), CNRS,
+ * France
+ *
+ * @date 2023/12/22
+ *
+ * Vol visualization using polyscope.
+ *
+ * This file is part of the DGtal library.
+ */
+#include <vector>
+#include <array>
+
+#include <DGtal/base/Common.h>
+#include <DGtal/helpers/StdDefs.h>
+#include <DGtal/helpers/Shortcuts.h>
+
+#include <polyscope/polyscope.h>
+#include "polyscope/surface_mesh.h"
+#include "polyscope/volume_mesh.h"
+#include "polyscope/point_cloud.h"
+
+#include "CLI11.hpp"
+
+// Using standard 3D digital space.
+using namespace DGtal;
+using namespace Z3i;
+typedef Shortcuts<Z3i::KSpace> SH3;
+
+
+/**
+ @page volscope volscope
+ 
+ @brief Volumetric file visualization using polyscope
+ 
+ @b Usage: volscope [options] --input  \<fileName\>
+ 
+ 
+ @b Allowed @b options @b are :
+ @code
+ Usage: ./volscope [OPTIONS] 1
+ 
+ Positionals:
+ 1 TEXT:FILE REQUIRED                  Input vol file.
+ 
+ Options:
+ -h,--help                             Print this help message and exit
+ -i,--input TEXT:FILE REQUIRED         Input vol file.
+ --volumetricMode                      Activate the volumetric mode instead of the isosurface visualization.
+ --point-cloud-only                    In the volumetric mode, visualize the vol file as a point cloud instead of an hex mesh (default: false)
+ --min INT                             For isosurface visualization and voxel filtering, specifies the threshold min (excluded) (default: 0).
+ --max INT                             For isosurface visualization and voxel filtering, specifies the threshold max (included) (default: 255).
+ @endcode
+**/
+
+int main(int argc, char**argv)
+{
+  // parse command line using CLI ----------------------------------------------
+  CLI::App app;
+  app.description("Vol file vizualization using polyscope");
+  
+  std::string inputFileName;
+  app.add_option("-i,--input,1", inputFileName, "Input vol file.")->required()->check(CLI::ExistingFile);
+  
+  bool volumetricMode = false;
+  app.add_flag("--volumetricMode",volumetricMode, "Activate the volumetric mode instead of the isosurface visualization.");
+
+  bool pclOnly = false;
+  app.add_flag("--point-cloud-only",pclOnly, "In the volumetric mode, visualize the vol file as a point cloud instead of an hex mesh (default: false)");
+  
+  int thresholdMin=0;
+  app.add_option("--min", thresholdMin, "For isosurface visualization and voxel filtering, specifies the threshold min (excluded) (default: 0).");
+  int thresholdMax=255;
+  app.add_option("--max", thresholdMax, "For isosurface visualization and voxel filtering, specifies the threshold max (included) (default: 255).");
+  
+  
+  app.get_formatter()->column_width(40);
+  CLI11_PARSE(app, argc, argv);
+  
+  polyscope::init();
```

Comment:
```suggestion 
  polyscope::options::programName = "volscope " + inputFileName +  " - (DGtalTools)";  
  polyscope::init();
```
(to sutomize windows title)